src/components/SessionTimeout/SessionTimeoutModal.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import Countdown from 'react-countdown';
+
+class SessionTimeoutModal extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    toggle = () => {
+         this.props.restartTimer();
+    }
+
+    renderer = ({hours, minutes, seconds, completed}) => {
+        if (completed) {
+            this.props.sessionTimedOut(false);
+            window.location='https://dev-mydata.kpmp.org/Shibboleth.sso/Logout?return=https://login.dev-mydata.kpmp.org/idp/profile/Logout'
```

Comment:
This makes sense, just a question on the link. Should this just be dev-mydata or should prod be handled as well?

Reply:
Yeah, it is currently hardcoded to dev-mydata because that is where we are testing and there are notes to update to mydata.kpmp.org as part of the release.